webrtc.html

```diff
@@ -2620,8 +2620,12 @@
           </section>
         </div>
       </section>
-      <section>
+      <section class="informative">
         <h3>Legacy Interface Extensions</h3>
+          <p>This section is non-normative; it specifies methods that can be
```

Comment:
Thanks @foolip (I was aware 'non-normative' is the the best fit here, I just wanted input on what I should say).
Is there another key word I could use instead of ```informative``` in 
```<section class="informative">``` ? I don't know what the valid ones are.